lib/buffer.js

```diff
@@ -1107,8 +1114,12 @@ Buffer.prototype.readUInt32BE = function readUInt32BE(offset, noAssert) {
 
 
 Buffer.prototype.readIntLE = function readIntLE(offset, byteLength, noAssert) {
+  if (!noAssert)
+    checkByteLength(byteLength);
+
   offset = offset >>> 0;
   byteLength = byteLength >>> 0;
```

Comment:
This is not necessary anymore, since the `byteLength` can only be in the range `x >= 1 && x <= 6`.

Reply:
I believe the subsequent changes I made now make this comment obsolete.